budgetportal/views.py

```diff
@@ -1,7 +1,7 @@
 import json
 import logging
 import urllib
-import urlparse
+import urllib.parse as urlparse
```

Comment:
I think this will be a bit confusing in a python 3 world - please update references to use urllib.parse or parse or something by its new name